### PR TITLE
Mapping candidate office titles to correct labels

### LINKED ISSUE #1269: Capitalization of senate and house
Senate and House should be capitalized in these headers:
screen shot 2016-06-13 at 11 20 06 pm
screen shot 2016-06-13 at 11 20 16 pm

But I cannot figure out where these suckers are hiding in the code. @ccostino or @LindsayYoung can one of you point me to the templates where these are, so I can fix them? :)

### PR DESCRIPTION
**Pull request by noahmanger**
Is this the correct casing @emileighoutlaw ?
image

image

image

Resolves https://github.com/18F/openFEC-web-app/issues/1269

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
Yep that's perfect!

File: static/js/pages/candidates-office.js [hunk 1]
```diff
@@ -21,10 +21,16 @@ var defaultSort = {
   house: 4
 };
 
+var officeTitleMap = {
+  president: 'president',
```
Comment (ccostino) [inline static/js/pages/candidates-office.js]: I noticed the other two titles are capitalized but not `president`, is this intentional?

Comment (ccostino) [inline static/js/pages/candidates-office.js]: Disregard, just caught up about this in Slack - this is intentional!

**Comment by ccostino** [comment]
Thanks, @noahmanger!

**Pull request merged**